{"id": "5340", "title": "Bug 5340 - Cancelling add exception breakpoint has no effect", "description": "", "OB": "When attempting to cancel the process of adding an exception breakpoint, the progress dialog closes immediately. However, the exception list continues to appear, disregarding the cancellation request. Instead, the process should either honor the cancellation request and stop adding the breakpoint, or disable the cancel button to prevent any further cancellation attempts. To accomplish this, when calling ProgressMonitorDialog.run, the value of the 'canceleable' parameter should be set to false.", "EB": "", "SR": ""}